scripts/geopmpy/plotter.py

```diff
@@ -20,7 +20,7 @@
 
 try:
     with open(os.devnull, 'w') as FNULL:
-        subprocess.check_call("python -c 'import matplotlib.pyplot'", stdout=FNULL, stderr=FNULL, shell=True)
+        subprocess.check_call(['python', '-c',  "'import matplotlib.pyplot'"], stdout=FNULL, stderr=FNULL)
```

Comment:
This probably always fails on systems that don't have an unversioned `python` symlink (vs `python3`).